src/ConnectDown.cpp

```diff
@@ -671,12 +671,7 @@ int connectdown(char *pfile, char *wfile, char *ad8file, char *outletshapefile,
 
     /* Create new file using this driver */
      hDSsh = OGR_Dr_CreateDataSource(driver, outletshapefile, NULL);
-     if (hDSsh == NULL)
-      {
-        printf("warning: Unable to create %s\n", outletshapefile);
-       // exit( 1 );
-	 }
-
+	 if( hDSsh  != NULL ) {
```

Comment:
I do not understand this.  Previously we were getting the warning due to null.  It seems like skipping doing anything is not going to fix it.